discord/http.py

```diff
@@ -745,7 +745,7 @@ def edit_custom_emoji(self, guild_id, emoji_id, *, name, roles=None, reason=None
         return self.request(r, json=payload, reason=reason)
 
     def get_all_integrations(self, guild_id):
-        r = Route('GET', '/guilds/{guild_id}/integrations', guild_id=guild_id)
+        r = Route('GET', '/guilds/{guild_id}/integrations?include_applications=True', guild_id=guild_id)
```

Comment:
```suggestion
        r = Route('GET', '/guilds/{guild_id}/integrations', guild_id=guild_id)
```

applications integrations are provided by default now.